host2str.c

```diff
@@ -713,7 +713,7 @@ ldns_rdf2buffer_str_loc(ldns_buffer *output, const ldns_rdf *rdf)
 	uint32_t longitude;
 	uint32_t latitude;
 	uint32_t altitude;
-	char northerness;
+	char northerliness;
```

Comment:
hopefully this is ok

Reply:
I do not think this would convey the meaning of the variable more clearly.
How about `latitude_hemisphere` and then `longitude_hemisphere` instead of `easterness`.

Reply:
That does sound better